cli/http_util.rs

```diff
@@ -79,6 +79,9 @@ fn resolve_url_from_location(base_url: &Url, location: &str) -> Url {
   }
 }
 
+// TODO(ry) HTTP headers are not unique key, value pairs. There may be more than
+// one header line with the same key. This should be changed to something like
+// Vec<(String, String)>
```

Comment:
https://github.com/denoland/deno/pull/4175/files#diff-82118b139b0bc05b336931d5fa0c4c76R119-R130

Reply:
Somehow links into diffs don't work for me. What are you linking to?

Reply:
```rust
let mut headers_: HashMap<String, String> = HashMap::new();
    let headers = response.headers();
    for key in headers.keys() {
      let key_str = key.to_string();
      let values = headers.get_all(key);
      let values_str = values
        .iter()
        .map(|e| e.to_str().unwrap().to_string())
        .collect::<Vec<String>>()
        .join(",");
      headers_.insert(key_str, values_str);
    }
```
This is done inside `fetch_once` - this guards against multiple same keys, but probably does concatenation in wrong manner

Reply:
Also `e.to_str().unwrap().to_string()` seems brittle

Reply:
It would be better to simply use `Vec<(String, String)>`... I will leave the todo